### PR TITLE
Add human recoloring

### PR DESCRIPTION
**Pull request by SpadesNeil**
Also gave Nevareans more lore while I was at it, and removed some unnecessary code.

Resolves https://github.com/VOREStation/VOREStation/issues/163

### CODE REVIEW

**Comment by SpadesNeil** [comment]
REVIEWMAHSHIT

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/datum/species/human
+	color_mult = 1
+	icobase = 'icons/mob/human_races/r_human_vr.dmi'
+	deform = 'icons/mob/human_races/r_def_human_vr.dmi'
```
Comment (Cameron653) [inline code/modules/mob/living/carbon/human/species/station/station_vr.dm]: These three lines could probably be included at the bottom of this file. species/human is already used down there to define the humans egg type.

**Comment by Cameron653** [comment]
I think you might need to add "has_skin_color" to the human appearance flags, too.

**Comment by SpadesNeil** [comment]
That should fix it.

**Comment by SpadesNeil** [comment]
Review iiiit

**Comment by Cameron653** [comment]
Looks good to me

**Pull request merged**